blue_modules/notifications.js

```
@@ -265,89 +275,63 @@ const _setPushToken = async token => {
+            onRegister: async token => {
+              console.debug('TOKEN:', token);
+              alreadyConfigured = true;
+              await _setPushToken(token);
+              resolve(true);
+            },
+            onNotification: async notification => {
+              const payload = { ...notification, ...notification.data };
+              if (notification.data && notification.data.data) {
+                Object.assign(payload, notification.data.data);
+              }
+              delete payload.data;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)